test/ecto_plus_one_test.exs

```diff
@@ -1,8 +1,52 @@
 defmodule EctoPlusOneTest do
   use ExUnit.Case
-  doctest EctoPlusOne
+  use EctoPlusOne.DataCase
 
-  test "greets the world" do
-    assert EctoPlusOne.hello() == :world
+  import Ecto.Query
+  import ExUnit.CaptureLog
+
+  alias EctoPlusOne.User
+  alias EctoPlusOne.Post
+
+  @repo Application.get_env(:ecto_plus_one, :repo)
```

Comment:
I would just `alias EctoPlusOne.Repo`...but not really important

Reply:
changing